CarShowRoom/customers/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+from django.utils import timezone
+from sellers.models import CarShowRoom, Car
+
+
+class Customer(models.Model):
+    first_name = models.CharField(max_length=40)
+    last_name = models.CharField(max_length=40)
+    email = models.EmailField()
```

Comment:
Customers, Dealers, Buyers and other potential user-based models should be inherited from django User model in order they can login to account (we need to implement user authentication as well in the future) + django user contain all of the necessary information list first name, etc -> https://docs.djangoproject.com/en/4.2/ref/contrib/auth/